pop-api/src/v0/fungibles/traits.rs

```diff
@@ -113,9 +121,12 @@ pub trait Psp22Mintable {
 pub trait Psp22Burnable {
 	/// Destroys `value` amount of tokens from `account`, reducing the total supply.
 	///
+	/// The selector for this message is `0xfc3c75d4` (first 4 bytes of
```

Comment:
Manually verified to be correct.

Reply:
<img width="1146" alt="Screenshot 2024-10-03 at 00 58 00" src="https://github.com/user-attachments/assets/c0085a62-1538-44fb-9875-24d7d1d63159">

Confirmed with https://www.shawntabrizi.com/substrate-js-utilities/